pkg/apis/install.relay.sh/v1alpha1/relaycore_types.go

```diff
@@ -344,6 +344,14 @@ type MetadataAPIConfig struct {
 }
 
 type VaultConfig struct {
+	// +kubebuilder:default="relaysh/relay-operator-vault-init:latest"
+	// +optional
+	Image string `json:"image"`
```

Comment:
You might want to either call this e.g. `InitializationImage` or nest it under an `Initialization` struct so people don't get confused that it's the image for Vault itself.

Reply:
Will do. This fits in with potentially adding in a default Vault deployment if desired/configured, which may be preferable to adding this externally in `relay`.